doc/r-plugin.txt

```diff
@@ -672,13 +671,13 @@ them.
 ------------------------------------------------------------------------------
 4.4. The Object Browser~
 
-You have to do <LocalLeader>ro to start the Object Browser. The Object Browser
-has two views: .GlobalEnv and Libraries. If you either press <Enter> or double
-click (GVim or Vim with 'mouse' set to "a") on the first line of the Object
-Browser it will toggle the view between the objects in .GlobalEnv and the
-currently loaded libraries. The Object Browser requires the |+clientserver|
-feature to be automatically updated and the |+conceal| feature to correctly
-align list items.
+You have to do <LocalLeader>ro to either start or update the Object Browser.
```

Comment:
The command `<LocalLeader>ro` no longer updates the Object Browser. It is updated automatically if the Vim feature 'clientserver' is enabled.